src/components/Button/Button.css

```diff
@@ -65,12 +66,26 @@
   color: $color__river;
 }
 
-.y-button:disabled,
 .y-button__state-disabled {
   opacity: 0.5;
   cursor: default;
 }
 
+.y-button__state-loading {
+  cursor: default;
+}
+
+.y-button__state-loading .y-button--contents {
```

Comment:
whenever possible we want to keep selectors at 1 class-level selector, for DOM performance and specificity

Reply:
Performance difference is insignificant, but agree wrt specificity.